ChangeLog

```diff
@@ -29,6 +29,11 @@ Bugfix
      by Brendan Shanks. Part of a fix for #992.
    * Fix compilation error when MBEDTLS_ARC4_C is disabled and
      MBEDTLS_CIPHER_NULL_CIPHER is enabled. Found by TrinityTonic in #1719.
+   * According to RFC 1779, commas in strings should be either in quotes,
```

Comment:
Style: Use imperative style and summarize the change in one sentence. Suggestion: `Adapt X.509 DN handling to conform to RFC 1779. Specifically, ...` or `Adapt X.509 DN handling to conform to RFC 1779: ...`